src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -474,6 +477,44 @@ class spell_dru_earthwarden : public AuraScript
     }
 };
 
+// 339 - Entangling Roots
+// 102359 - Mass Entanglement
+class spell_dru_entangling_roots : public AuraScript
+{
+    PrepareAuraScript(spell_dru_entangling_roots);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_DRUID_CURIOUS_BRAMBLEPATCH, SPELL_DRUID_ENTANGLING_ROOTS, SPELL_DRUID_MASS_ENTANGLEMENT });
+    }
+
+    bool CheckProc(ProcEventInfo& eventInfo)
+    {
+        if (SpellInfo const* spellInfo = eventInfo.GetSpellInfo())
```

Comment:
Why are you even checking the SpellInfo? its not like it can be null tho 👀

Reply:
It can, from autoattacks

Reply:
> 

ahhh right, damn, didn't think right now :)